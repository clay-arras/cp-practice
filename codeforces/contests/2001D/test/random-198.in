1
25
22 24 14 12 3 5 17 20 19 16 12 20 16 8 15 25 21 19 25 15 8 10 16 20 14
