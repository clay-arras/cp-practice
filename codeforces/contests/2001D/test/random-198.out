15
22 24 14 3 5 17 20 12 25 21 19 15 8 10 16
